doc/plotting/index.rst

```diff
@@ -363,3 +365,31 @@ NOTE: These functions works for only with matplotlib higher than 1.3.1.
                          :ref:`sphx_glr_auto_examples_01_plotting_plot_surf_stat_map.py`
 
 =====================   ===================================================================
+
+.. _interactive-surface-plotting:
+
+Surface plotting in a web browser
+---------------------------------
+
+Nilearn also has functions for making interactive surface plots that can be
+seen in a web browser.
+
+:func:`view_surf`: Surface plot using a surface map and a cortical mesh
+
+:func:`view_img_on_surf`: Surface plot using a 3d image.
```

Comment:
It would be nicer if you are more precise here. I think mostly a statistical maps image. Correct me if I am wrong.

Reply:
d should be capital 3d -> 3D